.github/workflows/release_build.yml

```diff
@@ -31,6 +31,21 @@ jobs:
       lane: review
       slack_thread_ts: ${{needs.start_slack_thread.outputs.thread_ts}}
       version: ${{ github.ref_name }}
+
+  notify_ios_failure:
+    name: Notify ios Failure
+    needs: [start_slack_thread, ios]
+    if: ${{ needs.ios.result == 'failure' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify Slack on iOS failure
+        run: |
+          curl -X POST \
+               -H 'Authorization: Bearer '"$SLACK_API_TOKEN" \
+               -H 'Content-type: application/json' \
+               -d '{"channel": "va-mobile-app", "text": "Version ${{ github.ref_name }} experienced a failure for iOS", "thread_ts": "${{ needs.start_slack_thread.outputs.thread_ts }}"}' \
```

Comment:
This message is a little vague. A link to the failed workflow would also be helpful.

I suggest something like:

`The release build failed for iOS vX.X.X. See <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|workflow run> for details.`